config/jobs/ppc64le-cloud/pod-utilities/podutilities-postsubmit.yaml

```diff
@@ -0,0 +1,100 @@
+postsubmits:
+  ppc64le-cloud/test-infra:
+    - name: postsubmit-podutilities-job
+      decorate: true
+      run_if_changed: 'images/pod-utilities'
+      branches:
+        - master
+      extra_refs:
+        - base_ref: v0.1.2
+          org: vmware-tanzu
+          repo: buildkit-cli-for-kubectl
+      spec:
+        serviceAccountName: build-img
+        containers:
+          - image: quay.io/powercloud/all-in-one:0.2
+            env:
+              - name: DOCKER_CONFIG
+                valueFrom:
+                  secretKeyRef:
+                    name: quay-powercloud-regcred
+                    key: .dockerconfigjson
+            volumeMounts:
+              - name: kubeconfig
+                mountPath: /etc/kubeconfig
+            command:
+              - /bin/bash
+            args:
+              - -c
+              - |
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+                pushd $GOPATH/src/github.com/vmware-tanzu/buildkit-cli-for-kubectl
+                make build install
+                popd
+                ca=$(base64 -w 0 /run/secrets/kubernetes.io/serviceaccount/ca.crt)
+                token=$(cat /run/secrets/kubernetes.io/serviceaccount/token)
+                namespace=$(cat /run/secrets/kubernetes.io/serviceaccount/namespace)
+                cat >kubeconfig <<EOL
+                apiVersion: v1
+                kind: Config
+                clusters:
+                - name: default-cluster
+                  cluster:
+                    certificate-authority-data: ${ca}
+                    server: https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT_HTTPS}
+                contexts:
+                - name: default-context
+                  context:
+                    cluster: default-cluster
+                    namespace: ${namespace}
+                    user: default-user
+                current-context: default-context
+                users:
+                - name: default-user
+                  user:
+                    token: ${token}
+                EOL
+                export KUBECONFIG=$(pwd)/kubeconfig
+                export utils=(clonerefs entrypoint initupload sidecar)
+                pushd images/pod-utilities
+                for i in ${utils[@]}
+                do
+                    kubectl build --push --registry-secret quay-powercloud-regcred --namespace image-builder --build-arg ARCH=amd64 --build-arg POD_UTILITY=$i -t quay.io/powercloud/$i:amd64-v20210126-f0baa5f3d9 -f Dockerfile ./
+                done
+                popd
```

Comment:
we aren't building the images for the amd64 arch, instead we are pulling them, retagging, and pushing into our repo, not sure if this logic really works!

Reply:
/cc @Rajalakshmi-Girish

Reply:
Yes as per the README available in pod-utilities folder, I have been only pulling, retagging, and pushing into our repo for amd64 arch.
I did not try building for amd64.

Reply:
If we can build it for amd64 then I don't mind build for amd64 platform as well.